src/dbSta/include/db_sta/dbSta.hh

```diff
@@ -83,6 +84,23 @@ class dbStaState : public sta::StaState
   dbSta* sta_ = nullptr;
 };
 
+enum CellFunction
+{
+  DATA,
+  CLOCK
+};
+
+class CellFunctionAnalyser
```

Comment:
Function is confusing as Liberty defines the cells function (eg Y=!A).  This doesn't apply to all cells, just buffers.  How about BufferUse ?